src/styles/What.css

```diff
@@ -84,3 +93,357 @@
+      margin-left: 40px;
+      box-shadow: 5px 5px 5px grey;
+    }
+    .Catch{
+        font-size: 15pt;
+    }
+    #catchDiv{
+        padding: 14px;
+    }
+    #title{
+      font-size: 20pt;
+    }
+}
+
+@media (max-width: 750px) and (min-width: 581px){
+    .Blurbs{
+        font-size: 13pt;
+        width: 57.5vw;
+    }
+    .LeftBlurbImages{
+        width: 75px;
+        height: 75px;
+    }
+    .LeftCircleDiv{
+        width: 85px;
+        height: 85px;
+    }
+    .RightBlurbImages{
+        width: 75px;
+        height: 75px;
+    }
+    .RightCircleDiv{
+        width: 85px;
+        height: 85px;
+    }
+    .fillLeftBlurbImages{
+        width: 100px;
+        height: 100px;
+    }
+    .fillRightBlurbImages{
+        width: 100px;
+        height: 100px;
+    }
+    .endingBlurbs{
+        font-size: 13pt;
+        width: 57.5vw;
+    }
+    #wwdRightBlurb {
+      width: 75px;
+      height: 75px;
+      border-radius: 50%;
+      margin-left: 40px;
+      box-shadow: 5px 5px 5px grey;
+    }
+    .Catch{
+        font-size: 20pt;
+    }
+    #catchDiv{
+        padding: 15px;
+    }
+    #title{
+      font-size: 22px;
+    }
+}
+
+@media (max-width: 1800px) and (min-width: 751px){
+    .Blurbs{
+        font-size: 15pt;
+        width: 60vw;
+    }
+    .endingBlurbs{
+        font-size: 15pt;
+        width: 60vw;
+      }
+      #wwdRightBlurb {
+        width: 100px;
+        height: 100px;
+        border-radius: 50%;
+        margin-left: 40px;
+        box-shadow: 5px 5px 5px grey;
+      }
+     .Catch{
+         font-size: 25pt;
+     }
+     #catchDiv{
+          padding: 17px;
+     }
+     #title{
+       font-size: 24pt;
+     }
+}
+
+@media (max-width: 1500px) and (min-width: 1000px){
+    .Blurbs{
+        font-size: 15pt;
+        width: 60vw;
+    }
+    .endingBlurbs {
+        font-size: 15pt;
+        width: 60vw;
+      }
+      #catchDiv{
+        padding: 18px;
+      }
+      #title{
+        font-size: 27pt;
+      }
+}
+
+@media (max-width: 1800px) and (min-width: 1501px){
+    .Blurbs{
+        font-size: 15pt;
+        width: 60vw;
+    }
+    .endingBlurbs {
+        font-size: 15pt;
+        width: 60vw;
+      }
+      #catchDiv{
+        padding: 19px;
+      }
+}
+
+
+@media (min-width: 1801px){
+    .Blurbs {
+        position: relative;
+        left: 5%;
+        background-color: #E8D3F7;
+        width: 73vw;
+        font-family: Raleway;
+        font-size: 18pt;
+        text-align: center;
+        font-weight: normal;
+        padding: 20px;
+        border-radius: 5px;
+        display: inline-block;
+    }
+
+    .endingBlurbs {
+          position: relative;
+          left: 5%;
+          background-color: #E8D3F7;
+          width: 73vw;
+          font-family: Raleway;
+          font-size: 18pt;
+          text-align: center;
+          font-weight: normal;
+          padding: 20px;
+          border-radius: 5px;
+          display: inline-block;
+      }
+}
```

Comment:
What.css: Code is clean. Good use of @media queries to cater to different devices with different resolutions, flex for auto-resizing, and removal of height.